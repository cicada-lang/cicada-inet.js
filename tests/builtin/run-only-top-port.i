require "../datatype/Nat.i"

begin
  one one add
  one one add
  run inspect
  swap inspect
  run inspect
end
