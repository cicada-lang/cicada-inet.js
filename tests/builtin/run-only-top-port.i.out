net_from_port (add1₂₅)-value!
  (add1₂₅)-prev value-(add1₂₃)
  (add1₂₃)-prev value-(zero₂₁)
end
net_from_port (add₁₀)-return
  (add₁₀)-target!value-(add1₂₂)
  (add₁₀)-addend value-(add1₂₁)
  (add1₂₂)-prev value-(zero₂₀)
  (add1₂₁)-prev value-(zero₁₉)
end
